src/microbe_stage/gui/PatchDetailsPanel.tscn

```diff
@@ -455,6 +454,31 @@ size_flags_vertical = 4
 mouse_filter = 2
 expand_mode = 1
 
+[node name="Radiation" type="HBoxContainer" parent="NormalContent/MarginContainer/ScrollContainer/VBoxContainer/PatchDetailsContainer/Compounds"]
+layout_mode = 2
+size_flags_horizontal = 3
+
+[node name="TextureRect" type="TextureRect" parent="NormalContent/MarginContainer/ScrollContainer/VBoxContainer/PatchDetailsContainer/Compounds/Radiation"]
+custom_minimum_size = Vector2(25, 25)
+layout_mode = 2
+size_flags_vertical = 4
+tooltip_text = "IRON"
+texture = ExtResource("22_00ty1")
+expand_mode = 1
+
+[node name="Label" type="Label" parent="NormalContent/MarginContainer/ScrollContainer/VBoxContainer/PatchDetailsContainer/Compounds/Radiation"]
+editor_description = "PLACEHOLDER"
+layout_mode = 2
+text = "5%"
```

Comment:
It looks like the real value doesn't have percentage suffix so I don't think the editor placeholder should either as it doesn't look like real data then.

Reply:
i know, I just copied it as other compounds also had percentage as placeholders

Reply:
Yeah, I expected as much but I didn't playtest this so I wasn't 100% sure that there wasn't a percentage sign added from somewhere in the game.